cypress/e2e/tests/migration/applicationinventory/analysis/custom-rules.test.ts

```diff
@@ -127,16 +127,16 @@ describe(["@tier2"], "Custom Rules in analyses", function () {
         tackleTestapp.verifyRulesNumber();
     });
 
-    it("Verify a file is not a valid XML", function () {
+    it.skip("Bug MTA-5779: Verify a file is not a valid XML", function () {
```

Comment:
the test name will change : 

it.skip("Bug MTA-5779: Verify a file is not a valid YAML",